pybaseball/analysis/trajectories/utils.py

```diff
@@ -0,0 +1,57 @@
+import numpy as np
+from functools import lru_cache
+from pybaseball.analysis.trajectories.unit_conversions import (
+    REVS_PER_MINUTE_TO_RADIANS_PER_SECOND,
+)
+
+
+@lru_cache(maxsize=64)
+def spin_components(
+        spin: float, spin_angle: float, launch_angle: float, launch_direction_angle: float) -> np.array:
+    """
+    given spin in revolutions per minute, spin angle in degrees,
+    launch angles in degrees, 
+    returns the x,y, z components of spin
+
+    :param spin:
+    :param spin_angle:
+    :return:
+    """
+
+    spin_angle_in_radians = np.deg2rad(spin_angle)
+    launch_angle_in_radians = np.deg2rad(launch_angle)
+    launch_direction_angle_in_radians = np.deg2rad(launch_direction_angle)
+
+    sidespin = spin * np.sin(spin_angle_in_radians)
+    backspin = spin * np.cos(spin_angle_in_radians)
+
+    wx = (
+        backspin * np.cos(launch_direction_angle_in_radians)
+        - sidespin * np.sin(launch_angle_in_radians) * np.sin(launch_direction_angle_in_radians)
+    ) * REVS_PER_MINUTE_TO_RADIANS_PER_SECOND
+    wy = (
+        - backspin * np.sin(launch_direction_angle_in_radians)
+        - sidespin * np.sin(launch_angle_in_radians) * np.cos(launch_direction_angle_in_radians)
+    ) * REVS_PER_MINUTE_TO_RADIANS_PER_SECOND
+    wz = sidespin * np.cos(launch_angle_in_radians) * REVS_PER_MINUTE_TO_RADIANS_PER_SECOND
```

Comment:
Can the RPM to Rad/sec conversion be applied just to the `spin` parameter once, and then not apply it elsewhere?
And what do you think about changing the name to `RPM_TO_RAD_SEC`?

Reply:
yes, good call. done.